cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,70 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
+describe("Student Registration page", () => {
+  const firstName = ["John", "Emma", "William", "Sophia", "Michael"][
+    Math.floor(Math.random() * 5)
+  ];
+  const lastNames = [
+    "Smith",
+    "Johnson",
+    "Williams",
+    "Jones",
+    "Brown",
+    "Davis",
+    "Miller",
+    "Wilson",
+    "Moore",
+    "Taylor",
+  ];
+  const randomLastName =
+    lastNames[Math.floor(Math.random() * lastNames.length)];
+  const email = `user${Math.floor(Math.random() * 10000)}@example.com`;
+  const phoneNumber =
+    "0" +
+    Math.floor(Math.random() * 1000000000)
+      .toString()
+      .padStart(9, "0");
+  const subjects = ["Math", "Science", "English", "History", "Art"];
+  const randomSubject = subjects[Math.floor(Math.random() * subjects.length)];
+
   before(() => {
-    
+    cy.visit("/");
   });
 
-  it('', () => {
-    
+  it("Registration", () => {
```

Comment:
I think that will be better to write "should allow to register". It's only my recommendation